packages/components/src/SvgIcon/SvgIcon.stories.tsx

```diff
@@ -41,119 +41,139 @@ type Args = React.ComponentProps<typeof SvgIcon> & {
   icon: keyof typeof allIcons;
 };
 
-const Template: Story<Args> = ({ icon, color, ...rest }) => {
-  const Icon = allIcons[icon];
-  const computedColor = color && ColorTokens[color];
-  return <Icon {...rest} color={computedColor} />;
-};
-
-const defaultArgs = {
-  icon: Object.keys(allIcons)[0] as keyof typeof allIcons,
-  role: "decorative" as const,
```

Comment:
Notice that `role` here isn't a valid prop (anymore), but TS failed to catch it.

Looks like object stories get us better types, and TS would not catch this.